packages/components/src/Menu/MenuGroup.tsx

```diff
@@ -66,7 +66,7 @@ const MenuGroupLayout: FC<MenuGroupProps> = ({
     <li className={className}>
       <MenuItemContext.Provider value={context}>
         {label && <MenuGroupLabel>{label}</MenuGroupLabel>}
-        <List>{children}</List>
+        <UnorderedList dense>{children}</UnorderedList>
```

Comment:
I think it might be better to just use `ul` here 

```suggestion
        <ul>{children}</ul>
```

Reply:
Resolved in 0e608bb